core/src/main/scala/runtime/implicits.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2017 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package freestyle
+package opscenter
+package runtime
+
+import freestyle.opscenter.model.Metric
+import org.http4s.{HttpService, _}
+import org.http4s.dsl._
+import org.http4s.dsl.Root
+import org.http4s.server.blaze._
+
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.io.{Source => SourceIO}
+
+object implicits {
+
+  implicit val metricsHandler: MetricsM.Handler[Future] = new MetricsM.Handler[Future] {
```

Comment:
These handlers have nothing in them that actually requires the `Future` api for what I can tell so they can be all parametric to `M: Monad` instead of `Future`. Use `Monad[M].pure` to lift already computed values to the context. If for whatever reason `Future` should stay already computed values should be lifted via `Future.successful`

Reply:
Thanks for the review. Finally I changed to Applicative and I am interpreting the program to Try